maquetado/scss/tech.scss

```
@@ -0,0 +1,49 @@
+.tech {
+  align-items: center;
+  background-color: $green-color;
```

Delete the `-color` isn't necessary
```suggestion
  background-color: $green;
```